Makefile

```diff
@@ -6,17 +6,13 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 # Use at your own risk. See README file for more details.
 
-JPEG_DIR ?= /opt/libjpeg-turbo
-JPEG_INCLUDE ?= $(JPEG_DIR)/include
-JPEG_LIB ?= $(JPEG_DIR)/lib`getconf LONG_BIT`
-
 CC   = gcc
 CFLAGS = -Wall -Wextra -O2
 GTK   = `pkg-config --libs --cflags gtk+-3.0` `pkg-config --libs x11`
 GTK  += `pkg-config --cflags --libs appindicator3-0.1`
 LIBAV = `pkg-config --libs --cflags libswscale libavutil`
 LIBS  =  -lspeex -lasound -lpthread -lm
-JPEG  = -I$(JPEG_INCLUDE) $(JPEG_LIB)/libturbojpeg.a
+JPEG  = -lturbojpeg
```

Comment:
The dynamic linking has been brought up many times before. This makes turbojpeg an install time dependency, and there is too much fragmentation across distros (for all packages). If/when the client is re-written, it can use the generic v4l2loopback and can potentially be vendored as a `.deb`.

Reply:
ah, this makes sense. It just worked so well on my test candidates, but they're rather current I admit.
I guess the only reason not to use upstream v4l2loopback is vendor fragmentation, too?
I'll revert the commit, if anyone wants to link dynamically they can just use `make JPEG=-lturbojpeg`.

Reply:
v4l2loopback used to be sparsely supported, but now I think its well supported. The droidcam fork mostly exists for historical and UX reasons.

turbojpeg is a challenge still.